docs/source/release-notes/2021-1-0-Eos.md

```diff
@@ -0,0 +1,166 @@
+# Alpha 2021.1.0 Eos Release Notes
+
+This version of Vircadia is a new release. It is **not** compatible with the 2020.3.0 Demeter protocol. This version was released on February 12th, 2021.
+
+## Interface (Codename Athena)
+
+The numbers at the end of each item are the PR numbers in the Vircadia [repo](https://github.com/vircadia/vircadia).
+
+#### General
+
+* UI/UX: The VR controls for regular and fullbody users has been overhauled. (#928)
+    1. Fix being unable to sit in the real world in VR without the avatar and viewpoint being popped up to a standing position (fixes https://github.com/vircadia/vircadia/issues/825). With the new option "Allow my avatar to stand", we can choose to have the avatar stand only when the user is standing. The default behavior is the same as before (always allowed to stand; allowed to lean only when user is standing), except that now, when foot trackers are used, the avatar is never made to stand artificially.
```

Comment:
Most of our PRs fix issues so starting to include them seems like a bad idea, unless we do a similar thing like on the PR numbers.
```suggestion
    1. Fix being unable to sit in the real world in VR without the avatar and viewpoint being popped up to a standing position. With the new option "Allow my avatar to stand", we can choose to have the avatar stand only when the user is standing. The default behavior is the same as before (always allowed to stand; allowed to lean only when user is standing), except that now, when foot trackers are used, the avatar is never made to stand artificially.
```

Reply:
Good point. The issue numbers should be in the PRs, anyway.